collect_app/src/test/java/org/odk/collect/android/activities/viewmodels/MainMenuViewModelTest.java

```diff
@@ -87,4 +105,64 @@ public void versionCommitDescription_whenDirtyCommit_returnsCommitCountAndSHAAnd
         MainMenuViewModel viewModel = new MainMenuViewModel(() -> "v1.14.0-181-ge51d004d4-dirty");
         assertThat(viewModel.getVersionCommitDescription(), equalTo("181-ge51d004d4-dirty"));
     }
+
+    @Test
+    public void when_editSavedFormButtonIsEnabledInSettings_shouldEditSavedFormButtonBeVisibleReturnTrue() {
```

Comment:
I might be missing something but I think your tests in `MainMenuButtonsVisibilityTest` cover everything? Was there a reason you felt like you needed tests for the `MainMenuViewModel` as well?

Reply:
Yeah that's right those tests were redundant, fixed.